src/pages/index.tsx

```diff
@@ -174,12 +170,52 @@ const PostponeSection = ({ nextPostponeRoute }) => (
     </p>
 
     <Link href={nextPostponeRoute} legacyBehavior>
+      <PostponeButton/>
+    </Link>
+  </>
+)
+const now = new Date();
```

Comment:
now by malo byt v kazdom komponente samostatne, teraz tie komponenty zavisia od niecoho co je mimo nich a neprichadza to cez props

Reply:
Predpokladam, ze toto je outdated, ak ano oznac to za Resolved prosim.